src/App/Cart/Service/CartCalculationService.php

```diff
@@ -50,12 +50,15 @@ public function calculateAllowedPackageTypes(PdkCart $cart): PackageTypeCollecti
      */
     public function calculateMailboxPercentage(PdkCart $cart): float
     {
-        if (! $cart->lines->every('product.mergedSettings.fitInMailbox', '>', 0)) {
+        if ($cart->lines->where('product.mergedSettings.fitInMailbox', 0)
+                ->count() > 0) {
             return INF;
         }
 
         return $cart->lines->reduce(static function ($carry, $line) {
-            return $carry + $line->quantity * (100.0 / ($line->product->mergedSettings->fitInMailbox ?: 1));
+            return $line->product->mergedSettings->fitInMailbox === -1
```

Comment:
Gebruik de constante voor -1, ik dacht `Tristate::INHERIT`

Reply:
Ik vind deze lange ternary operator lastig te lezen, maar geen reden tot afkeur, maar als je het met me eens bent, zou je de `fitInMailbox` in een variabele kunnen zetten (dan hoe je hem ook maar 1 keer op te halen), een early return maken als die ‘inherited’ is, en dan de berekening relatief transparant neerzetten.